vulnerability-classes/injection/SSRFInjection.bcheck

```diff
@@ -0,0 +1,21 @@
+metadata:
+    language: v1-beta
+    name: "SSRF in each insetpoint"
+    description: "Inserts a injection callback into each parameter to detect SSRF"
```

Comment:
Should this be `Insert an Collaborator address into each parameter to detect SSRF`